versions/3.0/karma/preloaded-ontologies/cyber-ontology.ttl

```diff
@@ -0,0 +1,151 @@
+@prefix memex: <http://schema.dig.isi.edu/ontology/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix schema: <http://schema.org/> .
+@prefix prov: <http://www.w3.org/ns/prov#> .
+
+#########################################################
+#
+# Classes
+#
+#########################################################
+
+memex:Vulnerability a owl:Class ;
+	rdfs:label "Vulnerability" ;
+	rdfs:subClassOf schema:Intangible ;
+	rdfs:comment """""" ;
+	.
+
+memex:CVSS a owl:Class ;
+	rdfs:label "CVSS" ;
+	rdfs:subClassOf schema:Intangible ;
+	rdfs:comment "CVSS represented as a structured object" ;
+	.
+
+memex:Exploit a owl:Class ;
+	rdfs:label "Exploit" ;
+	rdfs:subClassOf schema:CreativeWork ;
+	.
+
+#########################################################
+#
+# Properties
+#
+#########################################################
+
+# Vulnerability
+#
+
+memex:vulnerabilityOf a rdf:Property ;
+	rdfs:label "vulnerabilityOf" ;
+	schema:domainIncludes memex:Vulnerability ;
+	schema:rangeIncludes memex:SoftwareSystem ;
+	rdfs:comment "Specifies software contains this vulnerability" ;
+	.
+
+memex:hasVulnerability a rdf:Property ;
+	owl:inverseOf memex:vulnerabilityOf ;
+	.
```

Comment:
Need domainIncludes, rangeIncludes and comment field because we don't use owl reasoner.